packages/css-framework/src/contexts/_light.scss

```diff
@@ -1,190 +1,237 @@
-  
   // Breakpoints
   $__breakpoints: (
     root: (
       breakpoint: 0,
-      baseFontSize: 100%
+      baseFontSize: 100%,
     ),
     s: (
       breakpoint: 576px,
-      baseFontSize: 103%
+      baseFontSize: 103%,
     ),
     m: (
       breakpoint: 768px,
-      baseFontSize: 106%
-    ),
-    l: (
-      breakpoint: 992px,
-      baseFontSize: 109%
+      baseFontSize: 106%,
     ),
     xl: (
       breakpoint: 1200px,
-      baseFontSize: 112%
+      baseFontSize: 112%,
     ),
     xxl: (
       breakpoint: 1400px,
-      baseFontSize: 115%
+      baseFontSize: 115%,
     ),
     xxxl: (
       breakpoint: 1600px,
-      baseFontSize: 118%
-    )
-  ) !global;
-
+      baseFontSize: 118%,
+    ),
+  );
 
   // Set the context colours
+  // Todo - Review colour uses and remove neutral 00
   $__colors: (
-    neutral: (      
+    neutral: (
       white: #ffffff,
-      50:    #f7f9fb,
-      000:   #f0f4f8,
-      100:   #dae2ec,
-      200:   #bcccdc,
-      300:   #9eb3c8,
-      400:   #829ab1,
-      500:   #627d98,
-      600:   #496582,
-      700:   #324e68,
-      800:   #253b53,
-      900:   #102a43,
-      black: #091826
+      000: #f8fafc,
+      100: #e2e9ee,
+      200: #b8c7d2,
+      300: #748999,
+      400: #3e5667,
+      500: #233745,
+      600: #1c2d39,
+      700: #12202b,
+      800: #0c1720,
+      900: #0a141b,
+      black: #0a141b,
     ),
     primary: (
-      000: #e6f6ff,
-      100: #bbe3ff,
-      200: #7cc4fa,
-      300: #48a3f3,
-      400: #2286eb,
-      500: #0a67d2,
-      600: #0552b5,
-      700: #03449e,
-      800: #01337d,
-      900: #012159
+      000: #ecf8ff,
+      100: #ddf4ff,
+      200: #b7dff7,
+      300: #85c6f2,
+      400: #58aae9,
+      500: #3a8fdd,
+      600: #2a77c7,
+      700: #2661a7,
+      800: #274776,
+      900: #253b5b,
+    ),
+    altprimary: (
+      000: #e8f2ff,
+      100: #deebff,
+      200: #bbd5fe,
+      300: #99b7f9,
+      400: #7392f3,
+      500: #5b73e6,
+      600: #4e5cd3,
+      700: #4248b6,
+      800: #3b3985,
+      900: #343160,
     ),
     danger: (
-      000: #fef1f2,
-      100: #ffbdbd,
-      200: #ff9b9a,
-      300: #f76b6a,
-      400: #ef4e4d,
-      500: #e12d38,
-      600: #d01224,
-      700: #ab091f,
-      800: #8a041a,
-      900: #610315
+      000: #fff3f4,
+      100: #feeaec,
+      200: #fed1d1,
+      300: #fea9a9,
+      400: #fc7576,
+      500: #f44949,
+      600: #e13637,
+      700: #be2b2b,
+      800: #902727,
+      900: #692524,
     ),
     warning: (
-      000: #fffef4,
-      100: #fffbdb,
-      200: #fcf4ac,
-      300: #f5e473,
-      400: #edcb29,
-      500: #f2b449,
-      600: #ce892d,
-      700: #a46926,
-      800: #81491e,
-      900: #6b3012
+      000: #ffffee,
+      100: #fffddc,
+      200: #fefbb8,
+      300: #faed7e,
+      400: #f5db54,
+      500: #e8c242,
+      600: #cf9328,
+      700: #ae6d1d,
+      800: #8c4f17,
+      900: #693a12,
+    ),
+    altwarning: (
```

Comment:
Is `altwarning` the best use for purple? I don't have a good suggestion for another name though I'm afraid...

Reply:
I know. no idea 🤷‍♂

Reply:
info maybe.. or Deep !